litmus-portal/frontend/src/pages/AnalyticsPage/index.tsx

```diff
@@ -1,15 +1,69 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
why are these disables used?

Reply:
Its throwing error in using Node as interface and saying its unused although i have used it.. @arkajyotiMukherjee (error in eslint check)